pkg/registries/docker/metadata_v1.go

```diff
@@ -6,9 +6,8 @@ import (
 	"strings"
 
 	"github.com/docker/distribution/manifest/schema1"
-	"github.com/docker/docker/image"
-	"github.com/gogo/protobuf/types"
 	"github.com/opencontainers/go-digest"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
but then in code we will have `imageSpec.Image` instead of `v1.Image` and previously we have `image.V1Image` so I think `v1` is OK. We already have a `v1Image` variable name so I understand that in this context `v1` is important not exactly from where we import this struct.